rbac/server/api/roles.py

```diff
@@ -136,6 +218,28 @@
 
 
 @ROLES_BP.get("api/roles/<role_id>")
+@doc.summary("Get a role by its role id.")
+@doc.description("Get a role by its role id.")
+@doc.consumes({"head": str}, location="query")
+@doc.produces(
```

Comment:
@doc.produces() missing a few fields like link, head, and some additional data fields. Here's a sample payload:
```
{
  "data": {
    "remote_id": "",
    "members": [],
    "description": "adsfadf!",
    "id": "2d1b6d5e-9917-49aa-91c9-3abf836f54b8",
    "created_date": 1565103963,
    "start_block_num": 7,
    "metadata": {
      "sync_direction": "OUTBOUND"
    },
    "tasks": [],
    "name": "Michael's Role",
    "administrators": [
      "651b8eef-036a-453e-b05f-e54b65beea30"
    ],
    "owners": [
      "651b8eef-036a-453e-b05f-e54b65beea30"
    ],
    "proposals": [],
    "end_block_num": 9223372036854776000,
    "packs": []
  },
  "link": "http://localhost:8000/api/roles/2d1b6d5e-9917-49aa-91c9-3abf836f54b8/?head=b6a4a4ef06c49e36f97e2c5142fde0eece20d8a9ab41fd117c05751fb3ef721a44d4799d0a95f33a2de1aab89a82c7ed25beea03dc786c93d8480b45fa536431&start=None&limit=None",
  "head": "b6a4a4ef06c49e36f97e2c5142fde0eece20d8a9ab41fd117c05751fb3ef721a44d4799d0a95f33a2de1aab89a82c7ed25beea03dc786c93d8480b45fa536431"
}
```